pkg/operation/botanist/botanist.go

```diff
@@ -109,7 +109,16 @@ func (b *Botanist) computeRequiredExtensions() map[string]sets.String {
 		if worker.Machine.Image != nil {
 			machineImagesSet.Insert(string(worker.Machine.Image.Name))
 		}
+		if worker.CRI != nil && worker.CRI.ContainerRuntimes != nil {
```

Comment:
```suggestion
		if worker.CRI != nil {
```

not needed, you range over `nil`